game/scripts/vscripts/common/webapi.lua

```diff
@@ -171,7 +172,23 @@ function WebApi:AfterMatch(winnerTeam)
 		WebApi:Send("match/after", requestBody)
 	end
 end
-
+function WebApi:PatreonPlayerUpdate(updates)
+	local requestBody = {
+		customGame = WebApi.customGame,
+		matchId = isTesting and RandomInt(1, 10000000) or tonumber(tostring(GameRules:GetMatchID())),
+		mapName = GetMapName(),
+		players = {}
+	}
+	for key, value in pairs(updates) do
```

Comment:
Use more descriptive variable names, i.e.
```suggestion
	for playerId, patreonUpdate in pairs(updates) do
```